pull_request_package/lib/github_status_reporter.rb

```diff
@@ -3,22 +3,57 @@
 
 class GitHubStatusReporter
   include ActiveModel::Model
-  attr_accessor :client, :logger, :pull_request
+  attr_accessor :client, :logger, :package
   
   def report
-    client.create_status('openSUSE/open-build-service', 'f73658fa927b90a0a3d28b79866215210b856390', 'pending', options) if [:success, :failed].include?(state)
+    summary = self.summary
+    state = :success
+    if summary[:failure] > 0
+      state = :failure
+    elsif summary[:pending] > 0 or summary[:success] == 0
+      state = :pending
```

Comment:
General question: Do we want to set a pending state? Because I thought we talked that we only want to set failure or success but not pending? (But I'm fine with setting it for now and we can get rid of it depending of what the rest of the team thinks)

Reply:
I only wanted to avoid us blocking valid PRs while debugging - but in 'production' we truely want pending, so that contributors can watch the live log :)